server/middlewares/verifyToken.js

```diff
@@ -0,0 +1,34 @@
+import jwt from 'jsonwebtoken';
+import QueryHelpers from '../helpers/QueryHelpers';
+
+const verifyToken = async (req, res, next) => {
+  const token = req.headers['x-access-token'];
+  if (!token) {
+    return res.status(401)
+      .send({
+        status: 401,
+        error: 'You need a token to access this route',
+      });
+  }
+  try {
+    const decoded = await jwt.verify(token, process.env.SECRET);
+    const { rows } = await QueryHelpers.getAll('users', 'id', [decoded.userId]);
+    if (!rows[0]) {
+      return res.status(400)
+        .send({
+          status: 400,
```

Comment:
This is not the right status code, please correct this

Reply:
Would 401 suffice?